crates/matrix-sdk-crypto/src/olm/group_sessions/sender_data.rs

```diff
@@ -77,6 +77,21 @@ where
             let key = Ed25519PublicKey::from_slice(&buf).map_err(|e| de::Error::custom(&e))?;
             Ok(Box::new(key))
         }
+
+        fn visit_bytes<E>(self, v: &[u8]) -> Result<Self::Value, E>
+        where
+            E: de::Error,
+        {
+            if v.len() == Ed25519PublicKey::LENGTH {
+                let mut buf = [0u8; 32];
```

Comment:
We could use `[0u8; Ed25519PublicKey::LENGTH]` here too I think, to it's more explicit.